invenio_app_ils/document_requests/jsonresolvers/document_request_document.py

```diff
@@ -30,16 +30,17 @@ def jsonresolver_loader(url_map):
     @get_pid_or_default(default_value=dict())
     def get_document(document_pid):
         """Return the Document record."""
+        Document = current_app_ils.document_record_cls
         document = Document.get_record_by_pid(document_pid)
-        authors = []
-        for author in document.get("authors", []):
-            if "full_name" in author:
-                authors.append(author["full_name"])
-        return {
-            "pid": document.get("pid"),
-            "title": document.get("title", ""),
-            "authors": authors,
-        }
+        return pick(
+            document,
+            "authors",
```

Comment:
sorry if I lost it in the frontend PR, but I am wondering if changing the structure of the resolved authors field will not break the UI somewhere